Submodels/MTFLO_call.py

```
@@ -1,20 +1,183 @@
+Email: T.S.Vermeulen@student.tudelft.nl
+Student ID: 4995309
+Version: 1.0
+
+Changelog:
+- V1.0: Initial working version
 """
 
 import subprocess
 import os
 
 class MTFLO_call():
```

Ruff (0.8.2)

Unnecessary parentheses after class definition

Remove parentheses

(UP039)